src/components/cookies-banner/cookies-banner.directive.js

```diff
@@ -20,9 +20,13 @@ const directiveCtrl = ($scope, $cookies) => {
 	$scope.visible = true;
 	const cookieKey = 'cookiesBannerConfirmed';
 
+	// setting expireDate to 2 years ahead
+	const expireDate = new Date();
+	expireDate.setDate(expireDate.getDate() + 730);
+
 	$scope.clicked = () => {
 		$scope.visible = false;
-		$cookies.put(cookieKey, 'true');
+		$cookies.put(cookieKey, 'true', { 'expires': expireDate });
```

Comment:
Jenkins test fails on ESlint step. Once this line aligns to the ESlint rules, the tests should pass.